src/pages/DelegatoryValidator/StakingBar.tsx

```diff
@@ -81,7 +86,11 @@ export default function StakingBar({validator}: ValidatorStakingBarProps) {
           <Typography>Stake</Typography>
         </Button>
       </ListItem>
-      <StakeDialog handleDialogClose={handleClose} isDialogOpen={dialogOpen} />
+      <StakeDialog
+        handleDialogClose={handleClose}
+        isDialogOpen={dialogOpen}
```

Comment:
is the Dialog being used anywhere else on this file? if not, probably can more all Dialog related stuff to the `StakeDialog` file

Reply:
we will also have a `unstakedialog` and `transaction succeed` dialog. in the future will consolidate all the dialog UI to a single component file.